src/components/Datatable/ControlsModule/ColumnsControls/ColumnsControls.stories.tsx

```diff
@@ -18,41 +19,42 @@ export default {
   },
 } as Meta;
 
-export const Default: Story<ColumnsControlsProps> = (args) => {
+export const Default: Story<ColumnsControlsProps> = ({ isOpen, ...args }) => {
+  const [isShown, setIsShown] = useState(isOpen);
   useColumnsControls(action('onColumnOrderChange'), simpleColumns);
 
   return (
-    <ColumnsControls {...args}>
-      {(onClick) => (
-        <button type="button" onClick={onClick}>
-          Open Columns
-        </button>
-      )}
+    <ColumnsControls isOpen={isShown} {...args}>
+      <button type="button" onClick={() => setIsShown((state) => !state)}>
+        Open Columns
+      </button>
     </ColumnsControls>
   );
 };
 Default.args = {
-  defaultIsOpen: true,
+  isOpen: true,
   onApply: action('ColumnsOnApply'),
   onClose: action('ColumnsOnClose'),
   onOpen: action('ColumnsOnOpen'),
   onReset: action('ColumnsOnReset'),
 };
 
-export const WithCustomOrder: Story<ColumnsControlsProps> = (args) => {
+export const WithCustomOrder: Story<ColumnsControlsProps> = ({
+  isOpen,
+  ...args
+}) => {
+  const [isShown, setIsShown] = useState(isOpen);
```

Comment:
could you change the incoming story prop `isOpen` to `isDefaultOpen` or similar and then `isShown` to `isOpen`? it might be easier to understand :)

Reply:
so `isOpen` is property of `ColumnsControls` and in this case it's not only default but controls the dropdown itself so `isOpen` is valid property name IMO. What I can do is to rename `isShown` to `isDefaultOpen`, but...